diff --git a/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/textMateWorkerTokenizerController.js b/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/textMateWorkerTokenizerController.js
index c66251f..5c3d216 100644
--- a/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/textMateWorkerTokenizerController.js
+++ b/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/textMateWorkerTokenizerController.js
@@ -137,6 +137,9 @@ class TextMateWorkerTokenizerController extends Disposable {
             this._applyStateStackDiffFn = applyStateStackDiff;
             this._initialState = INITIAL;
         }
+        if (this._store.isDisposed) {
+            return;
+        }
         for (const d of stateDeltas) {
             let prevState = d.startLineNumber <= 1 ? this._initialState : this._states.getEndState(d.startLineNumber - 1);
             for (let i = 0; i < d.stateDeltas.length; i++) {
diff --git a/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/worker/textMateWorkerTokenizer.js b/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/worker/textMateWorkerTokenizer.js
index a71a031..d4f89f8 100644
--- a/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/worker/textMateWorkerTokenizer.js
+++ b/node_modules/@codingame/monaco-vscode-textmate-service-override/vscode/src/vs/workbench/services/textMate/browser/backgroundTokenization/worker/textMateWorkerTokenizer.js
@@ -98,5 +98,8 @@ class TextMateWorkerTokenizer extends MirrorTextModel {
             this._diffStateStacksRefEqFn = diffStateStacksRefEq;
         }
+        if (this._isDisposed) {
+            return;
+        }
         const startTime = ( new Date()).getTime();
         while (true) {
             let tokenizedLines = 0;
